gammapy/datasets/map.py

```diff
@@ -1344,6 +1344,29 @@ def stat_sum(self):
         else:
             return cash_sum_cython(counts.ravel(), npred.ravel()) + prior_stat_sum
 
+    def stat_sum_asimov(self, model_name=None):
```

Comment:
what happens if `model_name=None` ?

Reply:
It gets passed to the `npred_signal`, which in general reduces the list of evaluators used here to the ones of the model given.
```python
        if model_names is not None:
            if isinstance(model_names, str):
                model_names = [model_names]
            evaluators = {name: self.evaluators[name] for name in model_names}
```
only problem I see is that `model_names=[model_name]` in case of `model_name=None` is actually not None and the evaluators dict from above will be emtpy
not sure if this is the wanted behaviour